packages/nimbus/src/components/tooltip/tooltip.mdx

```diff
@@ -13,36 +11,123 @@ menu:
+- **Position tooltips appropriately:** Place tooltips so they don't obscure the element they're related to or other important UI elements. Consider the user's mouse path and avoid accidental triggers.
+- **Maintain accessibility:** Includes keyboard navigation and screen reader compatibility. Use ARIA attributes like aria-describedby to link the tooltip to its target element.
+- **Avoid overuse:** Don't use tooltips for every single UI element. Overuse can make the interface feel cluttered.
+- **Don't rely on tooltips for essential information:**  Crucial information (like error states) should never be exclusively conveyed through tooltips, as they are not always discoverable or accessible to all users. Use tooltips for supplementary or secondary details.
 
-All instances of `Tooltip` consist of two components, the `TooltipTrigger` and
-the `Tooltip`.
 
-The `TooltipTrigger` is required to wrap both the trigger element and the
-`Tooltip`.
+## Usage
+
+Tooltips are a valuable tool for providing concise, contextual information, but they should be used strategically and not as a replacement for clear and accessible design. Think carefully about the type of information you're presenting and choose the most appropriate UI element for the job.
+
+
+### Using tooltips for icons buttons
+
+Icons often rely on shared universal understanding. A tooltip provides a text label that removes any ambiguity and ensures users understand the button's function. This is especially important for less common or custom icons.
+
+> [!TIP]\
+> **Do**
+>
+> - Describe the icon’s action quickly and concisely.  
 
 ```jsx-live
-const App = () => (
-  <TooltipTrigger>
-    <Button>Trigger Element</Button>
-    <Tooltip>Tooltip</Tooltip>
-  </TooltipTrigger>
-  )
+const App = () => <Flex height="100px">
+  <img src="/images/tooltip/tooltip-guideline-do.png" alt="tooltip do guideline" />
+</Flex>
 ```
 
-## Tooltip Trigger Props
+> [!CAUTION]\
+> **Don't**
+>
+> - Do not describe the icon visually.
+> - Do not over explain the action to the user.
+> - Don’t overlap tooltips with the object it is clarifying.
+
+```jsx-live
+const App = () => <Flex height="100px">
+  <img src="/images/tooltip/tooltip-guideline-dont.png" alt="tooltip do guideline" />
+</Flex>
+```
 
-<PropTable id="TooltipTrigger" />
 
-## Tooltip Props
+
+### Specs
 
 <PropTable id="Tooltip" />
+
+### Accessibility
+
+Tooltip triggers must be focusable and hoverable in order to ensure that all users can activate them. When displayed, TooltipTrigger automatically associates the tooltip with the trigger element so that it is described by the tooltip content.
+
+Authors SHOULD ensure that elements with the <u>role</u> tooltip are referenced through the use of <u>aria-describedby</u> before or at the time the tooltip is displayed.
+
+### Accessibility Standards
+**1. Perceivable:**
```

Comment:
The prevailing suggestion here is to make each of these numbered lists a header (####)